tests/test_feature_weights.py

```diff
@@ -0,0 +1,46 @@
+"""
+Test cases for feature_utilities.py
+"""
+import pytest
+import numpy as np
+import pandas as pd
+
+import pancancer_evaluation.utilities.feature_utilities as feats
+
```

Comment:
Why does this file look nothing like test_feature_weight.ipynb? Am I missing something?

Reply:
Good catch! `test_feature_weight.ipynb` was a notebook I wrote to try out some testing methods, which I eventually made into `pytest` unit tests in the `tests` directory. So they're really pretty different files, not intended to just be a converted notebook like the other ones.

I actually didn't mean to commit `test_feature_weight.ipynb` - I'll delete it and leave the unit tests.